modules/transports/core/nhttp/src/test/java/org/apache/synapse/transport/certificatevalidation/OCSPVerifierTest.java

```diff
@@ -151,7 +151,7 @@ private OCSPResp generateOCSPResponse(OCSPReq request, X509CertificateHolder cer
 
                 RevokedStatus revokedStatus = new RevokedStatus(new Date(), CRLReason.privilegeWithdrawn);
                 Date nextUpdate = new Date(new Date().getTime() + TestConstants.NEXT_UPDATE_PERIOD);
-                basicOCSPRespBuilder.addResponse(certID, revokedStatus, nextUpdate, null);
+                basicOCSPRespBuilder.addResponse(certID, revokedStatus, nextUpdate, (Extensions)null);
```

Comment:
Formatting issue: There should be a space after ()

Reply:
Fixed in PR https://github.com/wso2/wso2-synapse/pull/1302